README.md

```diff
@@ -2,30 +2,45 @@
 
 A React Native Camera and Player for MediaPipe applications.
 
-## Installation
+* [Documentation](https://cdiddy77.github.io/react-native-mediapipe/)
 
+## Requirements
+* Requirements go here.
+
+## Features
+* 🎥 Video streaming from a react-native app to a media pipe
+* 🔍 AI Object Detection
+* 🧩 Works as a react-native component
+
+Install mediapipe from npm:
 ```sh
 npm install react-native-mediapipe
 ```
 
-## Usage
+## Example
 
 ```js
-import { MediapipeView } from "react-native-mediapipe";
+import { MediaPipe_Camera_ } from "react-native-mediapipe";
 
 // ...
 
 <MediapipeView color="tomato" />
 ```
 
+> Link to an example of this in use later.
+
 ## Contributing
 
 See the [contributing guide](CONTRIBUTING.md) to learn how to contribute to the repository and the development workflow.
 
+Join the [Community](https://discord.gg/ApuAzVnAaX) here! 
```

Comment:
please add a link to the documentation: https://cdiddy77.github.io/react-native-mediapipe/

Reply:
I added the documentation near the top since this one did that https://github.com/react-native-firebase/react-native/blob/master/README.md and it felt like something important to have near the top, should I move it down?